src/internal/modules/contact-entry/forms/select-account-type.js

```diff
@@ -0,0 +1,50 @@
+const { formFactory, fields } = require('shared/lib/forms');
+const Joi = require('@hapi/joi');
+
+const form = (request, accountTypeDefaultValue, personNameDefaultValue) => {
+  const { csrfToken } = request.view;
+  const { sessionKey } = request.query;
+
+  const f = formFactory('/contact-entry/new/account-type');
+
+  f.fields.push(fields.radio('accountType', {
+    errors: {
+      'any.required': {
+        message: 'Select an option'
```

Comment:
If the error message has not been provided we copy the page title: Select the account type